test/e2e/suites/unmanaged/unmanaged_functional_test.go

```diff
@@ -862,6 +722,37 @@ var _ = ginkgo.Context("[unmanaged] [functional]", func() {
 				AvailabilityZone:  "us-west-2a",
 			}, e2eCtx)
 			wlClusterInfra.CreateInfrastructure()
+
+			ginkgo.By("Creating VPC peerings")
+			cPeering, _ = shared.CreatePeering(e2eCtx, mgmtClusterName+"-"+wlClusterName, *mgmtClusterInfra.VPC.VpcId, *wlClusterInfra.VPC.VpcId)
+		})
+
+		ginkgo.JustAfterEach(func() {
```

Comment:
Let's add some comments to both to the test and the helpers for clarifying things like why we need JustAfter and JustBefore closures for this test specifically and what topology is created in CreateInfrastructure() method etc.

Reply:
Done